src/widgets/Footer/index.tsx

```diff
@@ -23,7 +24,7 @@ const StyledFooter = styled.footer`
 const Footer: React.FC = () => (
   <StyledFooter>
     {config.map((entry) => {
-      const attributes = entry.href.startsWith("http") ? { target: "blank", rel: "noopener noreferrer" } : {};
```

Comment:
you can use the external prop directly

Reply:
`<Link key={entry.href} href={entry.href} external={entry.href.startsWith("http")} {...attributes}>`